lib/lazy_object.rb

```diff
@@ -27,4 +27,17 @@ def __target_object__
   def method_missing(method_name, *args, &block)
     __target_object__.send(method_name, *args, &block)
   end
+
+  def present?
+    !blank?
+  end
+
+  def blank?
+    __target_object__.respond_to?(:empty?) ? !!__target_object__.empty? : !__target_object__
+  end
+
+  # Forwards method calls to the target object.
```

Comment:
Now .present? doesn't forward to target_object and called on LazyObject, that always returns true